doc/source/rllib/rllib-advanced-api.rst

```diff
@@ -0,0 +1,548 @@
+exploratory behavior, allowing e.g. turning off any exploration easily for
+evaluation purposes (see :ref:`CustomEvaluation`).
+
+The following are example excerpts from different Algorithms' configs
+(see rllib/algorithms/algorithm.py) to setup different exploration behaviors:
+
+.. code-block:: python
+
+    # All of the following configs go into Algorithm.config.
+
+    # 1) Switching *off* exploration by default.
+    # Behavior: Calling `compute_action(s)` without explicitly setting its `explore`
+    # param will result in no exploration.
+    # However, explicitly calling `compute_action(s)` with `explore=True` will
+    # still(!) result in exploration (per-call overrides default).
+    "explore": False,
+
+    # 2) Switching *on* exploration by default.
+    # Behavior: Calling `compute_action(s)` without explicitly setting its
+    # explore param will result in exploration.
+    # However, explicitly calling `compute_action(s)` with `explore=False`
+    # will result in no(!) exploration (per-call overrides default).
+    "explore": True,
+
+    # 3) Example exploration_config usages:
+    # a) DQN: see rllib/algorithms/dqn/dqn.py
+    "explore": True,
+    "exploration_config": {
+       # Exploration sub-class by name or full path to module+class
+       # (e.g. “ray.rllib.utils.exploration.epsilon_greedy.EpsilonGreedy”)
+       "type": "EpsilonGreedy",
+       # Parameters for the Exploration class' constructor:
+       "initial_epsilon": 1.0,
+       "final_epsilon": 0.02,
+       "epsilon_timesteps": 10000,  # Timesteps over which to anneal epsilon.
+    },
+
+    # b) DQN Soft-Q: In order to switch to Soft-Q exploration, do instead:
+    "explore": True,
+    "exploration_config": {
+       "type": "SoftQ",
+       # Parameters for the Exploration class' constructor:
+       "temperature": 1.0,
+    },
+
+    # c) All policy-gradient algos and SAC: see rllib/algorithms/algorithm.py
+    # Behavior: The algo samples stochastically from the
+    # model-parameterized distribution. This is the global Algorithm default
+    # setting defined in algorithm.py and used by all PG-type algos (plus SAC).
+    "explore": True,
+    "exploration_config": {
+       "type": "StochasticSampling",
+       "random_timesteps": 0,  # timesteps at beginning, over which to act uniformly randomly
+    },
+
+
+.. _CustomEvaluation:
+
+Customized Evaluation During Training
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+RLlib will report online training rewards, however in some cases you may want to compute
+rewards with different settings (e.g., with exploration turned off, or on a specific set
+of environment configurations). You can activate evaluating policies during training (``Algorithm.train()``) by setting
+the ``evaluation_interval`` to an int value (> 0) indicating every how many ``Algorithm.train()``
+calls an "evaluation step" is run:
+
+.. code-block:: python
+
+    # Run one evaluation step on every 3rd `Algorithm.train()` call.
+    {
```

Comment:
Same here. I do realize this is a lot of translations. Maybe do this in a follow-up PR.
But we should really not mention, give-as-example, or advertise using the old config dicts anymore.